akka-docs/rst/java/code/docs/actorlambda/ActorDocTest.java

```diff
@@ -160,7 +160,7 @@ private void receiveMsg3(Msg3 msg) {
     public void onReceive(Object msg) throws Exception {
       if (msg instanceof Msg1)
         receiveMsg1((Msg1) msg);
-      else if (msg instanceof Msg1)
+      else if (msg instanceof Msg2)
```

Comment:
yes, this is not `UntypedActor`, it just looks like such.
`OptimizedActor extends UntypedAbstractActor`